src/views/ApartmentApp/components/StudentApplication/components/ApplicantList/index.js

```diff
@@ -40,37 +39,13 @@ const ApplicantList = ({
 
   /**
    * Callback for apartment people search submission
-   * @param {String} theChosenOne Username for student
+   * @param {String} selectedUsername Username for student selected via the people search
    */
-  const handleSelection = (theChosenOne) => {
+  const handleSelection = (selectedUsername) => {
     // Make sure the chosen username was not null
-    if (theChosenOne) {
+    if (selectedUsername) {
       // Send the selected username to the parent component
-      onSearchSubmit(theChosenOne);
-    }
-  };
-
-  /**
-   * Callback for changing the application editor
-   * @param {StudentProfileInfo} profile The StudentProfileInfo object for the person who is to be made the application editor
-   */
-  const handleChangeEditor = (profile) => {
```

Comment:
Where did this and the function below go?

Reply:
Oh, are these now just handled in StudentApplication/index.js?

Reply:
Basically, all it did was set the onChange___ of the component to this function, and then this function would call onChangeEditor without adding or doing anything else, so you can just "cut out the middle man" and pass the callback from the ApplicantList prop directly to the callback in the ApplicantListItem component below

Evan pointed this out in other places, so I figured I should clean up any others that I noticed later

Reply:
When commenting on parts like this, it helps to select also the lines after the function name, so that it is easy to view the full context of what your comment is referring to

Reply:
Oh, gotcha! I will try to remember that next time,